src/main/java/org/verdictdb/jdbc41/VerdictStatement.java

```diff
@@ -42,10 +42,67 @@ public VerdictStatement(VerdictConnection jdbcConn, VerdictContext context) {
     this.executionContext = context.createNewExecutionContext();
   }
 
+  /**
+   * Check whether given sql contains 'bypass' keyword at the beginning
+   *
+   * @param sql original sql
+   * @return without 'bypass' keyword if the original sql begins with it. null otherwise.
+   */
+  private String checkBypass(String sql) {
+    if (sql.trim().toLowerCase().startsWith("bypass")) {
+      return sql.trim().substring(6);
+    }
+    return null;
+  }
+
+  private VerdictSingleResult executeAsIs(String sql) throws SQLException, VerdictDBDbmsException {
+    DbmsConnection dbmsConn = context.getConnection();
```

Comment:
Do we need this check? I guess a simple "dbmsConn.execute(sql)" could work?

Reply:
Argh. I got it. Git highlighted checkByPass() method for the comment.

I guess you are right. I just wanted to be pedantic and use the actual JDBC statements whenever possible, but it does seem redundant at the same time.